momaland/learning/continuous_ppo/wrappers.py

```diff
@@ -0,0 +1,65 @@
+"""Wrappers for training.
+
+Parallel only.
+
+TODO AEC.
+"""
+
+import os
+from typing import Optional
+
+import pandas as pd
+import pettingzoo
+from pettingzoo.utils.wrappers.base_parallel import BaseParallelWrapper
+
+
+class RecordEpisodeStatistics(BaseParallelWrapper):
```

Comment:
This wrapper can be moved to the normal wrappers, and then this file can be renamed utils :-)